sass/components/_dropdown.scss

```diff
@@ -81,5 +81,9 @@ body.keyboard-focused {
 }
 
 .dropdown-trigger {
+  &:focus {
+    outline: 0;
```

Comment:
Please see https://www.google.com/search?channel=fs&client=ubuntu&q=outline+0+bad why this is a bad practice to remove it in general.